src/main/java/uk/gov/hmcts/reform/sscs/service/ReferenceData.java

```diff
@@ -0,0 +1,14 @@
+package uk.gov.hmcts.reform.sscs.service;
+
+import lombok.Getter;
+import org.springframework.stereotype.Component;
+
+@Component
+@Getter
+public class ReferenceData {
```

Comment:
Optional: Could we name this `VenueReferenceData`?

Reply:
There are going to be more classes in here

Reply:
Yeah, my 116 classes

Reply:
Okay.. please no @TeaWithLucas I can't take any more.